packages/web-runtime/src/store/config.js

```diff
@@ -38,6 +38,7 @@ const state = {
     defaultExtension: 'files',
     homeFolder: '',
     disablePreviews: false,
+    resourceTableLazy: true,
```

Comment:
What do you think about giving this option a more generic name? When we add more view modes (next up: grid view), we could reuse the same option. What do you think about `displayResourcesLazy` or similar?